tests/templates/test_subroutines/test_qpe.py

```diff
@@ -397,6 +397,32 @@ def qpe_circuit():
 
         assert qml.math.isclose(qpe_circuit()[0], 1)  # pylint: disable=unsubscriptable-object
 
+    @pytest.mark.jax
+    def test_jit(self):
+        """Test if JIT works properly on the example in documentation"""
```

Comment:
Please be consistent and clear. "the example in documentation" is not the best phrase to put in a docstring.